pages/awards/index.js

```diff
@@ -1,64 +1,87 @@
 import React, { useState, useEffect } from 'react'; // React
+import axios from 'axios';
 import { useRouter } from 'next/router'; // Routing (with buttons)
 import Link from 'next/link'; // Next link
+import Loader from 'react-loader-spinner'; // Spinning loader
 import Layout from '@components/Layout'; // Layout wrapper
 import Navigation from '@containers/Navigation'; // Navigation state
 
+import { formatDropdownOptions } from '@components/Dropdown'; // Format dropdown options util
+import AwardModal from '@components/awards/AwardModal';
 import Button from '@components/Button'; // Button
 import Title from '@components/Title'; // Title
 import Input from '@components/Input'; // Input
 import Tabs from '@components/Tabs'; // Tabs
-import Modal from '@components/Modal'; // Modal
-import Dropdown from '@components/Dropdown'; // Dropdown
-import FilterDropdown from '@components/FilterDropdown'; // Filter Dropdown
+import FilterDropdown, { formatFilterDropdownOptions } from '@components/FilterDropdown'; // Filter Dropdown
 import Table from '@components/Table'; // Table
-import Pill from '@components/Pill'; // Pill
 import Pagination from '@components/Pagination'; // Pagination
 import BackArrow from '@assets/back-arrow.svg'; // Back arrow icon
 import Search from '@assets/search.svg'; // Search icon
 import ChevronDown from '@assets/chevron-down.svg'; // Chevron down icon
 import ChevronDownGrey from '@assets/chevron-down-grey.svg'; // Chevron down grey icon
 import DancerRedJump from '@assets/dancer-red-jump.svg'; // Jumping Dancer SVG
 import DancerYellowBlue from '@assets/dancer-yellow-blue.svg'; // Jumping Dancer SVG
-import styles from '@styles/pages/Performances.module.scss'; // Page styles
+import styles from '@styles/pages/Awards.module.scss'; // Page styles
 
-// Temp constants
-import data, { columns } from '../../data/mockAwards';
-
-const PAGE_SIZE = 6; // Rows per page
+const PAGE_SIZE = 15; // Rows per page
 
 // Get the active filters (list of column accessors) from an object of filter dropdown values
 const getActiveFilters = options => {
   return Object.keys(options).filter(option => options[option].selected);
 };
 
 // Remove key from object (returns new object)
-const removeKeyFromObject = (object, key) => {
-  // eslint-disable-next-line no-unused-vars
-  const { [key]: _, ...rest } = object;
-  return rest;
-};
+// const removeKeyFromObject = (object, key) => {
```

Comment:
Do we still need this commented code?

Reply:
This is for the filtering, I'll un-comment in my next change so figure might leave it in?